drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/main/java/org/drools/workbench/screens/dtablexls/client/handlers/NewDecisionTableXLSHandler.java

```diff
@@ -141,6 +142,7 @@ public void execute() {
                                      busyIndicatorView.hideBusyIndicator();
                                      presenter.complete();
                                      notifySuccess();
+                                     newResourceSuccessEvent.fire( new NewResourceSuccessEvent( path ) );
```

Comment:
I have to ask; why introduce a new ```NewResourceSuccessEvent``` whereas most other observers of new, updated and deleted resources etc use ```ResourceAddedEvent``` (raised from Uberfire's ```IOWatchService```)? The "Library" will not see new resources created externally and pushed into the Workbench repository (e.g ,```git clone...```, update, ```git push...```)